_posts/2019-07-21-introducing-rico.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  'Introducing Rico'
+author: markus
+featuredImage: 
+excerpt: 'Introduction to Rico, the application framework.'
+permalink: '/rico/2019/07/16/introducing-rico.html'
+categories: [Java, Jakarta EE, Java EE, Spring, Rico, Remoting, Monitoring, Security, Angular, WebComponents, JavaFX, Projector]
+header:
+  text: Introducing <span class="my-karakun">Rico<span>
+  image: post
+---
+
```

Comment:
is this linking by _every_ Rico really necessary?

Reply:
to put emphasis on it :)